src/fieri/app/models/supported_platforms_worker.rb

```diff
@@ -0,0 +1,49 @@
+require 'sidekiq'
+require 'net/http'
+
+class SupportedPlatformsWorker
+  include ::Sidekiq::Worker
+
+  attr_accessor :version_info, :cookbook_name
+
+  def perform(version_json, cookbook_name)
+    @version_info = JSON.parse(version_json)
+    @cookbook_name = cookbook_name
+
+    response = Net::HTTP.post_form(
+      URI.parse("#{ENV['FIERI_SUPERMARKET_ENDPOINT']}/api/v1/quality_metrics/supported_platforms_evaluation"),
+      fieri_key: ENV['FIERI_KEY'],
+      cookbook_name: cookbook_name,
+      cookbook_version: version_info['version'],
+      supported_platforms_failure: supported_platforms_failure?,
+      supported_platforms_feedback: supported_platforms_feedback
+    )
+
+    return if response.is_a? Net::HTTPSuccess
+    raise "Unable to POST Supported Platforms Evaluation of #{cookbook_name}: #{response.code} #{response.message}"
+  rescue => e
+    log_error(e)
+  end
```

Comment:
I modeled this rescue'n'log pattern from the `FoodcriticWorker`. So, yay for logged errors, but as-is the rescue swallows the exception and Sidekiq thinks the job succeeded and won't retry. I'm going for a "_yes_ this is how we want to make workers" goal here, if only on one of them and I think I want retries.

Thoughts about whether that's how we want to keep it or whether we should spec the exceptions to be raised so the job is attempted again?